pkg/cri/config/config.go

```diff
@@ -263,6 +263,8 @@ type PluginConfig struct {
 	// Log line longer than the limit will be split into multiple lines. Non-positive
 	// value means no limit.
 	MaxContainerLogLineSize int `toml:"max_container_log_line_size" json:"maxContainerLogSize"`
+	// LogReadBufSize is the size of the read buffer in bytes.
+	LogReadBufSize int `toml:"log_read_buf_size" json:"logReadBufSize"`
```

Comment:
If this is going to be an int (as opposed to something like `16KB`), then I'd suggest to change this to `LogReadBufBytes`